daemon/networkdriver/bridge/driver.go

```diff
@@ -230,17 +233,25 @@ func InitDriver(job *engine.Job) error {
 		}
 	}
 
+	if hairpinMode {
+		// Enable loopback adresses routing
+		sysPath := filepath.Join("/proc/sys/net/ipv4/conf", bridgeIface, "route_localnet")
```

Comment:
is there some reason you couldn't just test for this file and set the default that way?